### ISSUE TITLE
"My organization only" fails due to wrong authorize_url

### ISSUE THREAD

**Issue by tmjoen** [dwyl/elixir-auth-microsoft#7]
Hi and first of all thanks so much for this library! It came just when I needed it :)

I have setup an Azure account and followed the instructions, but when I set the `Supported account types` in the AD application setup to `Accounts in this organizational directory only (Default Directory only - Single tenant)` the sign-in link fails:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/10501945/198031414-635ac3f8-822d-4897-8ac3-31bf5613f6a8.png">

If I delete this application, and create a new one like in your guide (where everyone is allowed) the sign up link works!

My elixir app unfortunately requires me to limit to only the organizations tenant and prevent access from anyone with an AD account!

I can fix it by changing the `@authorize_url` here https://github.com/dwyl/elixir-auth-microsoft/blob/main/lib/elixir_auth_microsoft.ex#L8 to:

`https://login.microsoftonline.com/<my_tenant_id_here>/oauth2/v2.0/authorize`

So basically switching out `common` with my tenant's id.

**Comment by nelsonic** [comment]
Hey @tmjoen 👋 
Stoked you found it useful already. 🎉 
Props to @LuchoTurtle for the implementation.
We have the optional callback URL feature over in: https://github.com/dwyl/elixir-auth-google#different-callback-url
So we'll take a look at your PR now. 👌

**Comment by nelsonic** [comment]
Hi @tmjoen 👋 
@LuchoTurtle is publishing a **`new`** version of the package with your changes: #10 📦 🚀 
Please test when you can and give feedback. 🙏
